contracts/src/v0.8/shared/test/call/CallWithExactGas.t.sol

```diff
@@ -149,6 +152,54 @@ contract CallWithExactGas__callWithExactGas is CallWithExactGasSetup {
   }
 }
 
+contract CallWithExactGas_callWithExactGasAndConsumeAllGas is CallWithExactGasSetup {
+  function test_callWithExactGasAndConsumeAll() external {
+
+    vm.pauseGasMetering();
+
+    uint16 maxRetBytes = 0;
+    uint256 misc_gas_overhead = 128;
+
+    bytes4 functionSelector = bytes4(keccak256("consumeAllGas()"));
```

Comment:
prefer using a reference over a computed string value

Reply:
When you say a reference do you mean hard coding the function signature `0xe5430851`?